### PR TITLE
openrgb: Add version 0.6

### PR DESCRIPTION
**Pull request by sitiom**

### CODE REVIEW

File: bucket/openrgb.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.6",
+    "description": "Open source RGB lighting control that doesn't depend on manufacturer software.",
+    "homepage": "https://openrgb.org",
+    "license": "GPL-2.0-or-later",
```
Comment (Ash258) [inline bucket/openrgb.json:5]: Source? Where is clearly stated it is `or later`?

File: bucket/openrgb.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.6",
+    "description": "Open source RGB lighting control that doesn't depend on manufacturer software.",
```
Comment (Ash258) [inline bucket/openrgb.json:3]:
```suggestion
    "description": "RGB lighting control that does not depend on manufacturer software.",
```

File: bucket/openrgb.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "homepage": "https://openrgb.org",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
```
Comment (Ash258) [inline bucket/openrgb.json:6]: image

File: bucket/openrgb.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    "shortcuts": [
+        [
+            "OpenRGB.exe",
+            "OpenRGB"
+        ]
+    ],
+    "checkver": {
+        "url": "https://gitlab.com/api/v4/projects/10582521/releases",
+        "jp": "$[0].description",
+        "re": "https://openrgb.org/releases/release_(.+)/OpenRGB_.+_Windows_(?:32|64)_(?<commit>[0-9a-f]{7}).zip"
```
Comment (Ash258) [inline bucket/openrgb.json:27]:
```suggestion
        "url": "https://openrgb.org/releases.html",
        "regex": "/OpenRGB_([\\d.]+)_Windows_\\d+_(?<commit>[\\da-fA-F]{7})\\.zip"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/openrgb.json (hunk 1 above)
Comment (sitiom) [reply bucket/openrgb.json:5]: Didn't know what to actually put here. But I guess it is not `or later` unless specified.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**